var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -55,9 +55,9 @@ class Geant4(CMakePackage):
     depends_on('cmake@3.8:', type='build', when='@10.6.0:')
     depends_on('cmake@3.5:', type='build')
 
-    for _vers in ["11.0.0", "10.7.3", "10.7.2", "10.7.1", "10.7.0", "10.6.3",
-                  "10.6.2", "10.6.1", "10.6.0", "10.5.1", "10.4.3", "10.4.0",
-                  "10.3.3"]:
+    for _vers in ["11.0.1", "11.0.0", "10.7.3", "10.7.2", "10.7.1", "10.7.0",
+                  "10.6.3", "10.6.2", "10.6.1", "10.6.0", "10.5.1", "10.4.3",
+                  "10.4.0", "10.3.3"]:
```

Comment:
What we probably need is to change the latest data version to an open-ended range `11:`, then restrict the version range when new data are needed.

Also to avoid having to reshuffle this whole list at every new version, could you reorder as ascending?

Reply:
Thanks for the suggestion, that is indeed a bit more elegant. I made the open-ended range `11.0:`. This still covers the current use case but should break a bit earlier in case this is not handled when the next minor version increase is added.

Re-arranged the list to be in ascending order.